tools/data/mmit/preprocess_data.sh

```diff
@@ -7,8 +7,7 @@ fi
 
 cd ${DATA_DIR}
 
-wget -c https://www.dropbox.com/s/sz3yd1o0gf09amh/Multi_Moments_in_Time.zip?dl=0
-
+# Download the Multi_Moments_in_Time_Raw.zip here manually
 unzip Multi_Moments_in_Time_Raw.zip
 rm Multi_Moments_in_Time.zip
```

Comment:
BTW I feel that `rm` here is dangerous.

Say, if user happens to not have unzip installed on the machine, then no files were extracted, and the raw data is also gone.

Reply:
You are right. So I decide to remove the rm